content/.vuepress/config.js

```diff
@@ -63,6 +63,12 @@ module.exports = {
                         },
                     ]
                 },
+                {
+                    title: 'Recipes',
```

Comment:
we should land the admin runbooks than rebase on that... allowing us to put this under "runbooks" instead of recipes